featuretools/tests/entityset_tests/test_ww_es.py

```diff
@@ -286,11 +287,65 @@ def test_update_dataframe():
     assert es['table'].ww.schema == original_schema
 
 
-def test_add_last_time_indexes(es):
+def test_add_last_time_index(es):
     es.add_last_time_indexes(['products'])
 
     assert 'last_time_index' in es['products'].ww.metadata
 
+    assert es['products'].ww.metadata['last_time_index'] == 'last_time'
+    assert 'last_time' in es['products']
+    assert 'last_time_index' in es['products'].ww.semantic_tags['last_time']
+    assert isinstance(es['products'].ww.logical_types['last_time'], Datetime)
+
+
+def test_add_last_time_non_numeric_index(pd_es, ks_es, dask_es):
+    pd_es.add_last_time_indexes(['products'])
+    dask_es.add_last_time_indexes(['products'])
+    ks_es.add_last_time_indexes(['products'])
+
+    assert list(to_pandas(pd_es['products']['last_time']).sort_index()) == list(to_pandas(dask_es['products']['last_time']).sort_index())
+    assert list(to_pandas(pd_es['products']['last_time']).sort_index()) == list(to_pandas(ks_es['products']).sort_values('id')['last_time'])
+
+    assert pd_es['products'].ww.schema == dask_es['products'].ww.schema
+    assert pd_es['products'].ww.schema == ks_es['products'].ww.schema
```

Comment:
Because `products` is the only dataframe in the mock entityset that doesn't have a numeric index, there were some bugs that were only showing up when the koalas and dask underlying indices (which will be numeric and therefore often happened to exactly match the woodwork index column) couldn't possibly match. 

So this test is confirming that the results of add_last_time index actually match in that case between pandas, dask, and koalas. It also confirms that the schemas match between the cases, because we're reinitializing Woodwork on a new DataFrame for the dask and koalas cases, and I wanted to make sure that was maintained as well.

Reply:
Got it. Thanks. Maybe a short code comment in the test would be helpful to help us remember that detail?

Reply:
added a comment!